nix/flakeModule.nix

```diff
@@ -5,7 +5,9 @@ in
 {
   imports = [
     ./os-config.nix
+    ./home-config.nix
     ./aspects-config.nix
   ];
-  options.den = types.denOption;
+  options.den.hosts = types.hostsOption;
```

Comment:
nice!, please also update the readme examples that used `den.<system>` to use `den.hosts.<system>`